src/patternfly/components/AppLauncher/themes/dark/app-launcher.scss

```diff
@@ -10,6 +10,5 @@
     --pf-c-app-launcher__menu-item--m-favorite__menu-item--m-action--Color: var(--pf-global--palette--gold-300);
     --pf-c-app-launcher__menu-item--m-favorite__menu-item--m-action--hover--Color: var(--pf-global--palette--gold-100);
     --pf-c-app-launcher__group-title--FontSize: var(--pf-global--FontSize--xs);
```

Comment:
you shouldn't need this since it's the default

```suggestion
```

Reply:
Updated